### ISSUE TITLE
[Feature Request] Default audio language

### ISSUE THREAD

**Issue by business-tomcat** [Dis90/plugin.video.discoveryplus#101]
Hi!

I really like this plugin! :+1:
The original discovery plus app is a pain in the a... ;)

There is only one downside: my favorite tv shows are always played in english, but I want the german audio track as default.
It is possible to create a configuration setting for the default audio language?

Thx and greetings!

WORKAROUND: press and hold the up key on the remote while playing a stream and Kodi will switch through audio streams. Same with down key for subtitles.

**Comment by Dis90** [comment]
Sorry there wont be any updates to this addon. Discovery+ is shutting down and they already canceled my subscription.

**Comment by business-tomcat** [comment]
Hi  @Dis90 

Thank you for your response!
This is completely new to me that Discovery+ is shutting down. I couldn't find any clues for this for Germany.
Quite the contrary, Discovery+ is now the only possibility to watch Eurosport 1 and 2 with their VODs, because the old official Eurosport app has been stopped in the middle of the year.
So I believe, german kodi users will continue to use your plugin :)

Perhaps I will make a fork of the repo and try to understand how it works and continue your work.

PS: If anybody find this thread, there is another solution: just long press the up button on the remote and kodi switches through possible audio streams. So it's very simple and fast to get the german stream.

**Comment by Dis90** [comment]
Interesting if they are't shutting it down on Germany.

Did you try Kodi settings -> Player -> Language -> Preferred audio language ?

This addon currently doesn't do any modification to video manifest. If you want to do modifications to it you basically need to use some proxy.

**Comment by vitusson** [comment]
Being from Germany too, also a heavy user of this addon, I first heard about the "shutdown" i.e moving into "Max" in this thread, So I'm not still sure about how this will play out in the long run for us German subscribers. I've got no message about a change right now, but that might mean nothing in these day and age. Companies come and go, customers have to suck
As for the language, I fear there's no easy solution. I'm subscribed for cycling races. And not every race has German audio stream. If I watch in the browser on ,y PC, the language I choosed is somehow saved and the next time I turn it on, it stays there. But if there's is no German stream , it falls back to no language or English as default. As it does on the addon if I watch in my bedroom via Kodi. 
Long story short: from my experience, that's all done on Discovery's side. not in addon or Kodi lang settings…and how it will continue, we're all depend on some companies decisions, none of which they will tell us about…
Having said tat: I've no solution and also no clue at all if I'll be able to watch my sport in 2025
THUMBSTHEFUCKDOWN
Thank you four workm no thanks for Discoverorwhatevertheownerwillbenexttime

**Comment by Dis90** [comment]
Here's some information about closing https://en.wikipedia.org/wiki/Discovery%2B
There is no mention of Germany, but I think that the service will end there as well during the next year

**Comment by business-tomcat** [comment]
Really interesting @Dis90 ... 
Best thing we can do is just to wait and see what's happening.
As @vitusson said, things will probably move to Max or something else and we need a new app :)

I will close this thread as resolved.
